docs/src/manual/solutions.md

```diff
@@ -32,6 +32,29 @@ Subject to
  y[b] ≤ 1.0
 ```
 
+## Solutions summary  
+[`show_solution_summary`](@ref) can be used for checking the summary of the optimization solutions.
+
+```jldoctest solutions
+julia> show_solution_summary(model)
+* Solver:GLPK
+
+* Status
+  Termination status : OPTIMAL
+  Primal status      : FEASIBLE_POINT
+  Dual status        : FEASIBLE_POINT
+  Message from the solver:
+  "Solution is optimal"
+
+* Candidate solution
+  Objective value : -205.14285714285714
+  Objective bound : Inf
+  Dual objective value : -205.1428571428571
+
+* Work counters
+  Solve time (sec)   : 7.414817810058594e-5
```

Comment:
We need to write a doctest filter for this: https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#Filtering-Doctests. Something like: `filter = r"[0-9]+.[0-9]+"`

(I'm going to suggest a related change below.)

Reply:
I added the filter.